worldengine/world.py

```diff
@@ -416,22 +415,10 @@ def start_mountain_th(self):
         return self.elevation['thresholds'][2][1]
 
     def max_elevation(self):
-        max_el = None
-        for y in range(self.height):
-            for x in range(self.width):
-                el = self.elevation['data'][y][x]
-                if max_el is None or el > max_el:
-                    max_el = el
-        return max_el
+        return self.elevation['data'].max()
 
     def min_elevation(self):
-        min_el = None
-        for y in range(self.height):
-            for x in range(self.width):
-                el = self.elevation['data'][y][x]
-                if min_el is None or el < min_el:
-                    min_el = el
-        return min_el
+        return self.elevation['data'].min()
```

Comment:
replace all calls to min_elevation to the .min(), saves us time calling a function just to get to numpy's builtin.